images/k8s-rdma-shared-dev-plugin-config-map.yaml

```diff
@@ -14,7 +14,10 @@ data:
            {
              "resourceName": "hca_shared_devices_b",
              "rdmaHcaMax": 500,
-             "devices": ["ib3", "ib4"]
+             "selectors": {
```

Comment:
can you change to the equivalent to what was here before ? (i.e use ifNames selector)